src/app/editor-common/components/key-value-component/key-value-input.component.ts

```diff
@@ -0,0 +1,129 @@
+import {Component, forwardRef, Input} from "@angular/core";
+import {
+    NG_VALUE_ACCESSOR,
+    ControlValueAccessor,
+    FormGroup,
+    FormBuilder,
+    NG_VALIDATORS,
+    FormControl
+} from "@angular/forms";
+import {ExpressionModel} from "cwlts/models/d2sb";
+import {ComponentBase} from "../../../components/common/component-base";
+
+require("./key-value-input.component.scss");
+
+@Component({
+    selector: "[ct-key-value-input]",
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => KeyValueInputComponent),
+            multi: true
+        },
+        {
+            provide: NG_VALIDATORS,
+            useExisting: forwardRef(() => KeyValueInputComponent),
+            multi: true
+        }
+    ],
+    template: `
+
+            <div class="key-container col-sm-5 input-group">
```

Comment:
Remove input-group, adds inconsistent style to class input field by setting left border-radiuses to 0.